frontend/workflows/dynamodb/src/update-capacity.tsx

```diff
@@ -0,0 +1,254 @@
+import React from "react";
+import type { clutch as IClutch } from "@clutch-sh/api";
+import {
+  Button,
+  ButtonGroup,
+  CheckboxPanel,
+  client,
+  MetadataTable,
+  Resolver,
+  Table,
+  TableRow,
+  useWizardContext,
+} from "@clutch-sh/core";
+import { useDataLayout } from "@clutch-sh/data-layout";
+import type { WizardChild } from "@clutch-sh/wizard";
+import { Wizard, WizardStep } from "@clutch-sh/wizard";
+import styled from "@emotion/styled";
+import _ from "lodash";
+import { number } from "yup";
+
+import type { ResolverChild, WorkflowProps } from "./index";
+
+const Container = styled.div({
+  display: "flex",
+});
```

Comment:
you could use mui's `Box` for this instead of a custom component if desired

Reply:
Swapped over to `Box` 👍